docs/src/main/sphinx/release/release-439.md

```diff
@@ -0,0 +1,46 @@
+# Release 439 (15 Feb 2024)
+
+## General
+
+* Fix failure when setting session properties for a catalog with a `.` in its
+  name. ({issue}`20474`)
+* Fix potential out-of-memory query failures when using the experimental scheduler. ({issue}`20694`)
+* Fix potential performance regression when dynamic filters are not applied. ({issue}`20709`)
+
+## BigQuery connector
+
+* Fix failure when pushing down predicates into BigQuery views. ({issue}`20627`)
+
+## Delta Lake connector
+
+* Improve performance when reading data by adding support for
+  [caching data on local storage](/object-storage/file-system-cache). ({issue}`18719`)
+* Fix potential crash when reading corrupted Snappy data. ({issue}`20631`)
+
+## Hive connector
+
+* Improve performance when reading data by adding support for
```

Comment:
Does it improve performance? Also, given the entry below, this one is confusing. I would combine them and say something like:

“remove support for enabling caching with xxx. This has been replaced with <link to caching data on local storage>”

Reply:
Testing revealed 10% and more in terms of performance improvements..